packages/apps/shopify-app-remix/src/server/authenticate/webhooks/types.ts

```diff
@@ -140,25 +139,10 @@ export interface WebhookContextWithoutSession<Topics = string | number | symbol>
   admin: undefined;
 }
 
-export interface LegacyWebhookAdminApiContext<
-  Resources extends ShopifyRestResources,
-> {
-  /** A REST client. */
-  rest: InstanceType<Shopify['clients']['Rest']> & Resources;
-  /** A GraphQL client. */
-  graphql: InstanceType<Shopify['clients']['Graphql']>;
-}
-
-export type WebhookAdminContext<
-  Future extends FutureFlagOptions,
-  Resources extends ShopifyRestResources,
-> =
-  FeatureEnabled<Future, 'v3_webhookAdminContext'> extends true
-    ? AdminApiContext<Resources>
-    : LegacyWebhookAdminApiContext<Resources>;
+export type WebhookAdminContext<Resources extends ShopifyRestResources> =
```

Comment:
Do we still need this type? Could we just use AdminApiContext instead?

Reply:
Yes, we can return `AdminApiContext`, I wasn't sure if we wanted to keep the more specific type name.

Reply:
Personally, I'd prefer not to alias types unless we had to for backward compatibility.